ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/internal/pojoadapter/UncleBlockAdapter.java

```diff
@@ -15,13 +15,15 @@
 package org.hyperledger.besu.ethereum.api.graphql.internal.pojoadapter;
 
 import org.hyperledger.besu.ethereum.core.BlockHeader;
-import org.hyperledger.besu.util.uint.UInt256;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
 
-@SuppressWarnings("unused") // reflected by GraphQL
+import org.apache.tuweni.units.bigints.UInt256;
+
+@SuppressWarnings("unused")
+// reflected by GraphQL
```

Comment:
Did the formatter forcefully move that comment block?  If it can't share the same line as the suppress warnings it should come before it.

Reply:
fixed!